documentation/release_notes.markdown

```diff
@@ -32,7 +32,7 @@ Released 17 September 2019
 
 ### Resolved issues
 
-- Puppet Server can now be reloaded and run with multiple JRuby instances when running under Java 11. [SERVER-2423](https://tickets.puppetlabs.com/browse/SERVER-2423)
+- Puppet Server can now be reloaded and run with multiple JRuby instances when running under Java 11. Note this change affects the packaging of Puppet Server and if running from source users will need to add the facter.jar provided by the puppet-agent package to the classpath when starting Puppet Server using Java. [SERVER-2423](https://tickets.puppetlabs.com/browse/SERVER-2423)
```

Comment:
This is a terrible sentence. I am bad at words.

Reply:
It's not too bad, I'd just add a little punctuation. Something like:
"Note this change affects the packaging of Puppet Server: if running from source, users will need to add the `facter.jar` provided by the puppet-agent package to the Java classpath when starting Puppet Server."

Reply:
I would suggest:

This change affects the packaging of Puppet Server. If you are running Puppet Server from source, you must add `facter.jar`, provided by the `puppet-agent` package, to the classpath when starting Puppet Server with Java.